docs/how-to-provision-using-dps.md

```diff
@@ -413,52 +437,69 @@ namespace Samples.AdtIothub
+Next, you'll need to set environment variables in your function app from earlier, containing the reference to the Azure Digital Twins instance you've created and the event hub. If you used the the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md)) the first setting will already be configured.
 
-You will need the following values from when you set up your instance. 
-If you need to gather this value again, use the link below to the corresponding section in the setup article for finding them in the [Azure portal](https://portal.azure.com).
-* Azure Digital Twins instance **_host name_** ([find in portal](../articles/digital-twins/how-to-set-up-instance-portal.md#verify-success-and-collect-important-values))
+You will need the following values from when you set up your instance.
+If you need to gather these values again, use the links below to the corresponding sections in the setup article for finding them in the [Azure portal](https://portal.azure.com).
+* Azure Digital Twins instance **_host name_** ([find in portal](how-to-set-up-instance-portal.md#verify-success-and-collect-important-values))
 * Azure Event Hubs connection string **_connection string_** ([find in portal](../event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal))
 
+Add the setting with this command:
+
 ```azurecli-interactive
 az functionapp config appsettings set --settings "ADT_SERVICE_URL=https://<Azure Digital Twins instance _host name_>" -g <resource group> -n <your App Service (function app) name>
 ```
 
-Next you will need to configure the function environment variable for connecting to the newly created event hub. 
+Next, you will need to configure the function environment variable for connecting to the newly created event hub.
 
 ```azurecli-interactive
 az functionapp config appsettings set --settings "EVENTHUB_CONNECTIONSTRING=<Event Hubs SAS connection string Listen>" -g <resource group> -n <your App Service (function app) name>
 ```
 
-Additionally review the section 'Assign permissions to the function app' in the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end#assign-permissions-to-the-function-app)) for configuring Function app permissions and role assignment with Managed Identity.
+Additionally, review the section *Assign permissions to the function app* in the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end#assign-permissions-to-the-function-app)) for configuring function app permissions and role assignment with Managed Identity.
```

Comment:
[Duplicate of comment on similar content in line 241] We can be more crisp about the ask-- Are we asking the reader to actually visit that article and perform the steps in it right now? Or just dropping a reference in case they want more information?

Reply:
Agree. The reader should actually visit the article and perform the steps (if not already done), because it is the only way to get access to the ADT instance and to get the sample running.